bin/cylc-graph

```diff
@@ -204,6 +207,9 @@ def main():
 
     (options, args) = parser.parse_args()
 
+    if no_x:
+        sys.exit(1)
```

Comment:
`--help` causes the script to exit before this point... and I just confirmed this really is the case by faking a gtk ImportError.